modules/oe_whitelabel_extra_project/oe_whitelabel_extra_project.post_update.php

```diff
@@ -27,5 +28,29 @@ function oe_whitelabel_extra_project_post_update_00001(): void {
  */
 function oe_whitelabel_extra_project_post_update_00002(): void {
   \Drupal::service('module_installer')->install(['field_group']);
-  ConfigImporter::importSingle('module', 'oe_whitelabel_extra_project', '/config/post_updates/00002_field_group', 'core.entity_view_display.node.oe_project.oe_w_content_banner');
+  $default_settings = \Drupal::service('plugin.manager.field_group.formatters')->getDefaultSettings('html_element', 'view');
+
+  $field_group = [
+    'children' => [],
+    'parent_name' => '',
+    'label' => 'Action bar',
+    'format_type' => 'html_element',
+    'format_settings' => $default_settings,
+    'region' => 'content',
+    'weight' => 20,
+  ];
+
+  $display = EntityViewDisplay::load('node.oe_project.oe_w_content_banner');
+
+  // If no display was found, we bail out.
+  if (!isset($display)) {
+    return;
+  }
+  // If there is a group_action_bar, we bail out.
+  if ($display->getThirdPartySetting('field_group', 'group_action_bar')) {
+    return;
+  }
+
+  $display->setThirdPartySetting('field_group', 'group_action_bar', $field_group);
+  $display->save();
```

Comment:
Last we return:
`Action bar field group was added to the event content banner view display.`